templates/apex-0.1/src/__ALGO__.py

```diff
@@ -1,9 +1,61 @@
 import Algorithmia
+import torch as th
 import apex
 
 """
-This package set comes preinstalled with Nvidia Apex 0.1.
+This package set comes preinstalled with Nvidia Apex 0.1
+
+Example Input:
+{
+    "matrix_a": [[0, 1], [1, 0]],
+    "matrix_b": [[25, 25], [11, 11]]
+}
+
+Expected Output:
+{
+    "product": [[11, 11], [25, 25]]
+}
 """
 
+device = th.device('cuda:0' if th.cuda.is_available() else 'cpu')
```

Comment:
if we're gonna test gpu support, we shouldn't default to CPU - we should make sure that GPU support works, otherwise there's not much reason to do testing